api/imapsync/read

```diff
@@ -128,9 +129,10 @@ if ($cmd eq 'list') {
     my $name = $input->{'name'};
     my $Port = $input->{'Port'};
     my $Security = $input->{'Security'};
-    my $exclude = $input->{'Exclude'};
-    $exclude =~ s/,/|/g;
-    $exclude = '|'.$exclude  if ($exclude ne '');
+    my @exclusion = split(',',$input->{'Exclude'});
+    push (@exclusion, '^Trash') if $input->{'TrashSync'} eq 'disabled';
+    my $exclude = join ('|', @exclusion) || '';
+    $exclude = '|'. $exclude  if ($exclude ne '');
```

Comment:
I think the previous implementation was more readable. The patch can be smaller if we just append a line like this after L133:

```perl
$exclude .= '|^Trash' if $input->{'TrashSync'} eq 'disabled';
```

What do you think?

Reply:
need to be tested because 

`||` this will exclude all
`toto|titi|` same

It is hard to have the good behaviour

Reply:
done good idea